app/models/main.py

```diff
@@ -1832,6 +1837,276 @@ def serialize(self):
+        # named as such to be explicit that this includes incomplete "process outcomes"
+        backref="process_outcomes_all",
+    )
+
+    brief_response = db.relationship(
+        "BriefResponse",
+        foreign_keys=brief_response_id,
+        # named as such to be explicit that this includes incomplete "process outcomes"
+        backref="process_outcomes_all",
+    )
+    brief = db.relationship(
+        "Brief",
+        foreign_keys=brief_id,
+        # named as such to be explicit that this includes incomplete "process outcomes"
+        backref="process_outcomes_all",
+    )
+
+    def serialize(self):
+        return {
+            "id": self.external_id,
+            "completed": self.completed_at is not None,
+            "completedAt": self.completed_at.strftime(DATETIME_FORMAT) if self.completed_at is not None else None,
+            "result": self.result,
+            **(
+                {
+                    "resultOfDirectAward": {
+                        "projectId": self.direct_award_project.external_id,
+                        **(
+                            {
+                                "searchId": self.direct_award_search_id,
+                                "archivedServiceId": self.direct_award_archived_service_id,
+                                "serviceId": self.direct_award_archived_service.service_id,
+                            } if self.result == "awarded" else {}
+                        ),
+                    },
+                } if self.direct_award_project_id is not None else {}
+            ),
+            **(
+                {
+                    "resultOfFurtherCompetition": {
+                        "briefId": self.brief_id,
+                        **(
+                            {"briefResponseId": self.brief_response_id} if self.result == "awarded" else {}
+                        )
+                    },
+                } if self.brief_id is not None else {}
+            ),
+            **(
+                {
+                    "award": {
+                        "startDate": self.start_date and self.start_date.strftime(DATE_FORMAT),
+                        "endDate": self.end_date and self.end_date.strftime(DATE_FORMAT),
+                        "awardingOrganisationName": self.awarding_organisation_name,
+                        # don't risk json representing this as a float
+                        "awardValue": str(self.award_value) if self.award_value is not None else None,
+                    },
+                } if self.result == "awarded" else {}
+            ),
+        }
+
+    @validates(
+        "start_date",
+        "end_date",
+        "awarding_organisation_name",
+        "award_value",
+    )
+    def _validates_data_complete_if_completed_at(self, key, value):
+        if key == "award_value":
+            if value is not None and value < 0:
+                raise ValidationError(f"{key} must be greater than or equal to zero, got {value!r}")
```

Comment:
If value is `None` then it will pass the validation, is that what you were going for?